services/press-fault-data-simulator-service/app/routers/simulator_router.py

```
@@ -0,0 +1,80 @@
+                data=status,
+            )
+        else:
+            return SimulationResponse(
+                success=False,
+                message="시뮬레이션 시작에 실패했습니다. (이미 실행 중이거나 API 서버 연결 실패)",
+                data={},
+            )
+
+    except Exception as e:
+        system_log.error(f"시뮬레이션 시작 API 오류: {str(e)}")
+        raise HTTPException(status_code=500, detail=f"내부 서버 오류: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)